src/App.tsx

```diff
@@ -1,17 +1,125 @@
-import React from 'react';
-import './App.css';
+import React, { useState } from 'react';
+import './App.scss';
 
 import users from './api/users';
+import todos from './api/todos';
+import { Todo } from './components/Types/Types';
+import { TodoList } from './components/TodoList/TodoList';
+
+const preparedTodos: Todo[] = todos.map(todo => {
+  return {
+    ...todo,
+    user: users.find(user => user.id === todo.userId),
+  };
+});
 
 const App: React.FC = () => {
+  const [todo, setTodo] = useState<Todo[]>([...preparedTodos]);
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [titleError, setTitleError] = useState(false);
+  const [userIdError, setUserIdError] = useState(false);
+
+  const addTodo = () => {
+    setTodo((prev) => ([
+      ...prev,
+      {
+        userId,
+        id: prev.length + 1,
+        title,
+        user: users.find(user => user.id === userId),
+        completed: false,
+      },
+    ]));
+  };
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    setTitleError(!title);
+
+    if (userId === 0) {
+      setUserIdError(true);
+    } else {
+      setUserIdError(false);
+    }
+
+    if (!title || userIdError) {
+      return;
+    }
+
+    setTitle('');
+    setUserId(-1);
```

Comment:
select element doesn't reset after submitting a new todo

<img width="186" alt="Screenshot 2022-04-21 at 14 03 56" src="https://user-images.githubusercontent.com/57598825/164444888-aaaa042e-ab3b-4ce5-9ad6-4d23f69d5496.png">

Reply:
maybe reset to 0?